pom.xml

```diff
@@ -10,43 +10,196 @@
 		<url>www.informaticisenzafrontiere.org</url>
 	</organization>
 
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.2.4</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+
 	<groupId>org.isf</groupId>
 	<artifactId>OH-core</artifactId>
+	<packaging>jar</packaging>
 	<version>1.14.0-SNAPSHOT</version>
```

Comment:
strange, this is supposed to be 1.14.1-SNAPSHOT now

Reply:
done